### ISSUE TITLE
[Feature] When pasting diagram, place it next to selection

### ISSUE THREAD

**Issue by jvdwetering** [tikzit/tikzit#60]
If you are zoomed in and paste something (ctrl+V), it appears at the edge of the diagram, not at the edge of the part that is visible. This gets annoying when dealing with large diagrams.
It would be nice to at least have the option to do one of the following: past it at the side of your selection, past it at the position of your mouse, past it at the side of the visual part of the diagram.

**Comment by aymannel** [comment]
Agreed. I use TikZit far more than ZXLive thanks to the greater number of keyboard shortcuts + configurability. The option to paste 'inplace' would be greatly appreciated!
